### PR TITLE
Library

### PR DESCRIPTION
**Pull request by 1v4n4**
This simple JavaScript app allows users to create a list of books, by entering their title, author, and the number of pages in a modal form. Inputs can't be blank.
Users can mark books as read/not read via the Status button. This value is false by default.
They can delete the book via the Delete button. 
Instead of an index, the app has a title-author uniqueness check.  Users can't enter the book with the same title-author values.
App stores data in local storage.

!Important
I disabled the linter for these mistakes: 
- 40:21  error    Expected parentheses around arrow function argument
- 56:10  error    Expected to return a value at the end of function 'bookCheck'   
- 64:41  error    Expected to return a value at the end of arrow function 
- 117:7   error    Expected an assignment or function call and instead saw an expression  

if that is a problem, please include instructions on how to fix them, because I tried everything I could without success.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <tr>
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td>${book.pages}</td>
+      <td class='reads'>${book.read}</td>
+      <td><button class="changeStatusBtn btn btn-secondary">Change status</button></td>
+      <td><button class="deleteBtn btn btn-danger">Delete book</button></td>
+    </tr>
+  `;
+}
```
Comment (trekab) [inline index.js:29]: :thinking: It appears that your approach differs from the expected implementation. Please note that you are expected to loop through the books array to display each book. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/books_render_loop.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @1v4n4! So far so good! :+1: I think you made good use of the `Book` constructor and form validation to reject empty book submissions. However, there are a few changes I'd like you to work on to finally complete this project. These include:
- [x] The __new book form__ should have a provision for the user to mark the book's read status. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/new_book_button_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))
- [x] __[Optional]__ Implement data persistence by saving, retrieving, and updating the library books using [the localstorage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/localstorage_stretch_requirement.png) from [Odin Project](https://www.theodinproject.com/courses/javascript/lessons/library))
- [ ] Please refer to the code comment below about the logic of displaying the books.

_Please make the necessary changes and submit another code review request._
__[Treasure Kabareebe](https://trekab.github.io)__

File: index.js (hunk 1 above)
Comment (1v4n4) [reply index.js:29]: well, now I needed that method to display books from local storage, so it is done

**Comment by mimipeshy** [comment]
## STATUS: APPROVED 🥳 👯 
Hello team @1v4n4 👋🏾 👋🏾 congratulations on making it this far in this milestone 🥇 

#### Highlights
- You have worked on all changes requested by the previous TSE ✔️ 

With that, I approve of this project 😄 

Regards
__Peris Ndanu __
[GitHub ](https://github.com/mimipeshy)| [LinkedIn ](https://www.linkedin.com/in/peris-ndanu/)| [Twitter](https://twitter.com/pygirl254)

**Pull request merged**